src/Payments.sol

```diff
@@ -1821,8 +1822,7 @@ contract Payments is ReentrancyGuard {
         auction.startPrice = uint88(auctionPrice);
         auction.startTime = uint168(block.timestamp);
 
-        (bool success,) = BURN_ADDRESS.call{value: msg.value}("");
-        require(success, Errors.NativeTransferFailed(BURN_ADDRESS, msg.value));
+        require(msg.value.burn(), Errors.NativeTransferFailed(BURN_ADDRESS, msg.value));
```

Comment:
Similar comment as in PDP, explicit call will be less "magical" and more readable.